community/tools/ra-data-hasura/src/index.js

```diff
@@ -32,6 +30,32 @@ export default (serverEndpoint, headers) => {
       throw new Error(JSON.stringify({'error': 'Invalid table/schema resource'}));
     }
 
+    return {schema: schema, tableName: tableName};
+
+  };
+
+  const getPrimaryKey = (resource) => {
+
+    let primaryKey = 'id';
+
+    if (config && config['primaryKey'][resource]) {
```

Comment:
Nitpick:

```suggestion
    if (config && config.primaryKey[resource]) {
```

Again, this is a stylistic thing but considered a good practice. Refer to https://github.com/airbnb/javascript#properties--dot.